fairlearn/datasets/_fetch_acsincome.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import pathlib
+
+import numpy as np
+import pandas as pd
+from sklearn.datasets import fetch_openml
+from ._constants import _DOWNLOAD_DIRECTORY_NAME
+
+
+def fetch_acsincome(*, cache=True, data_home=None,
+                    as_frame=False, return_X_y=False,
+                    states=None,
+                    ):
+    """Load the ACS Income dataset.
+
+    Download it if necessary.
+
+    ==============   ==============
+    Samples total           1664500
+    Dimensionality               10
+    Features                   real
+    Target                     real
+    ==============   ==============
+
+    Source: Paper: Ding et al. (2021) [1]_
+            and corresponding repository https://github.com/zykls/folktables/
+
+    .. versionadded:: 0.7.1
+
+    Parameters
+    ----------
+    cache : bool, default=True
+        Whether to cache downloaded datasets using joblib.
+
+    data_home : str, default=None
+        Specify another download and cache folder for the datasets.
+        By default, all fairlearn data is stored in '~/.fairlearn-data'
+        subfolders.
+
+    as_frame : bool, default=False
+        If True, the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string or categorical). The target is
+        a pandas DataFrame or Series depending on the number of target_columns.
+        The Bunch will contain a ``frame`` attribute with the target and the
+        data. If ``return_X_y`` is True, then ``(data, target)`` will be pandas
+        DataFrames or Series as describe above.
+
+    return_X_y : bool, default=False
+        If True, returns ``(data.data, data.target)`` instead of a Bunch
+        object.
+
+    states: list, default=None
+        List containing two letter (capitalized) state abbreviations.
+        If None, data from all 50 US states, including Puerto Rico, will be returned.
+        The state abbreviations and codes can be found on page 1 of the data dictionary at ACS PUMS
+        https://www2.census.gov/programs-surveys/acs/tech_docs/pums/data_dict/PUMS_Data_Dictionary_2018.pdf
+        and are reproduced in the `_STATE_CODES` dictionary below.
```

Comment:
"and are reproduced in the `_STATE_CODES` dictionary below."
users reading this on the docs page won't know what "dictionary below" means.  Perhaps you can just remove that part, or say "defined in the dictionary used in the source code of this method".

Reply:
I'll just remove that phrase since the linked pdf already provides the codes and abbreviations.